README.spanish.md

```diff
@@ -583,17 +620,17 @@ Todo lo de encima hubiera devuelto `false` si se hubiera usado `===`
 
 **De lo contrario:** Es probable que tu aplicación utilice solo el 25% de sus recursos disponibles (!) O incluso menos. Ten en cuenta que un servidor típico tiene 4 núcleos de CPU o más, la implementación ingenua de Node.js utiliza solo 1 (¡incluso utilizando servicios PaaS como AWS beanstalk!)
 
-🔗 [**Leer más: Utilize all CPU cores**](/sections/production/utilizecpu.spanish.md)
+🔗 [**Leer más: Utiliza todos los núcleos de la CPU**](./sections/production/utilizecpu.spanish.md)
 
 <br/><br/>
 
-## ![✔] 5.7. Crear un "punto final de mantenimiento"
+## ![✔] 5.7. Crear un "end point de mantenimiento"
```

Comment:
punto_final is the literal translation but doesn't have the same meaning, and also most people use it untranslated